src/app/(dashboard)/agents/page.tsx

```
@@ -0,0 +1,25 @@
+import React, { Suspense } from "react";
+
+const AgentsPage = async () => {
+  const queryClient = getQueryClient();
+  void queryClient.prefetchQuery(trpc.agents.getMany.queryOptions());
+
+  return (
+    <HydrationBoundary state={dehydrate(queryClient)}>
+      <Suspense fallback={<AgentsViewLoading />}>
+        <ErrorBoundary fallback={<AgentsViewError />}>
+          <AgentsView />;
+        </ErrorBoundary>
```

Biome (1.9.4)

[error] There is a suspicious semicolon in the JSX element.

This is usually the result of a typo or some refactor gone wrong.
Remove the semicolon, or move it inside a JSX element.

(lint/suspicious/noSuspiciousSemicolonInJsx)